### ISSUE TITLE
7.x version is not working in bundlephobia

### ISSUE THREAD

**Issue by TheBit** [probablyup/markdown-to-jsx#451]
https://bundlephobia.com/package/markdown-to-jsx@7.1.7
> MinifyError
We could not minify one of the source files in this package or its dependencies. Please verify if the contents of 0 can be minified using [terser](https://try.terser.org/).

```
Stacktrace
{ "name": "SyntaxError", "message": "Unexpected token: keyword (export)", "filename": "0", "line": 1, "col": 13941, "pos": 13941 }
```
While previous major is working fine: https://bundlephobia.com/package/markdown-to-jsx@6.11.4

**Comment by probablyup** [comment]
Yeah dunno what the issue is, they don’t really give any useful feedback

**Comment by probablyup** [comment]
Hmm sounds like they aren’t parsing the package.json bundling flags well and are assuming cjs
